dspace/config/spring/api/external-openaire.xml

```diff
@@ -10,17 +14,75 @@
 		<property name="clientId" value="${openaire.token.clientId}"/>
 		<property name="clientSecret" value="${openaire.token.clientSecret}"/>
     </bean>
-	<bean
-		class="org.dspace.external.provider.impl.OpenAIREFundingDataProvider"
-		init-method="init">
+
+	<bean class="org.dspace.external.provider.impl.OpenAIREFundingDataProvider" init-method="init">
 		<property name="sourceIdentifier" value="openAIREFunding" />
 		<property name="connector" ref="openAIRERestConnector" />
+		<property name="metadataFields" ref="mapOfmetadata"/>
 		<property name="supportedEntityTypes">
 			<list>
 				<value>Project</value>
 			</list>
 		</property>
 	</bean>
-	
-</beans>
 
+    <util:map id="mapOfmetadata"
+        map-class="java.util.HashMap" key-type="java.lang.String" value-type="org.dspace.importer.external.metadatamapping.MetadataFieldConfig">
+        <entry key="awardNumber" value-ref="openaireAwardNumber" />
+        <entry key="fundingStream" value-ref="openaireFundingStream" />
+        <entry key="awardURI" value-ref="openaireAwardURI" />
+        <entry key="funderName" value-ref="openaireFunderName" />
+        <entry key="funderIdentifier" value-ref="openaireFunderIdentifier" />
+        <entry key="dcTitle" value-ref="openaireTitle" />
+        <entry key="titleAlternative" value-ref="openaireTitleAlternative" />
+        <entry key="dcIdentifier" value-ref="openaireIdentifier" />
+        <entry key="coverageSpatial" value-ref="openaireSpatial" />
+        <entry key="dcDescription" value-ref="openaireDescription" />
```

Comment:
Same here, `dcDescription` doesn't appear to be used or defined in `OpenAIREFundingDataProvider`.  So, I don't think this setting does anything right now.